sql/derived_tables/instance_administrative_notes.sql

```diff
@@ -0,0 +1,25 @@
+-- This derived table extracts administrative notes from the instance
+-- record.
+
+DROP TABLE IF EXISTS instance_administrative_notes;
+
+CREATE TABLE instance_administrative_notes AS
+SELECT
+    i.id AS instance_id,
+    i.hrid AS instance_hrid,
+    admin_note.data #>> '{}' AS administrative_note,
+    admin_note.ordinality AS administrative_note_ordinality
+FROM
+    inventory_instances AS i
+    CROSS JOIN LATERAL json_array_elements(json_extract_path(data, 'administrativeNotes'))
```

Comment:
It's my understanding that we want to avoid the use of camel case. Here you have administrativeNotes. Could you change it to administrative_notes or administrativenotes?

Reply:
It is true that identifiers, such as names of tables and columns, should be in lowercase and should use underscores to separate words, rather than using camel case; this is documented in the contributing guidelines at https://github.com/folio-org/folio-analytics/blob/main/CONTRIBUTING.md#8-naming-things .  However, `'administrativeNotes'` is not an identifier but a string (note the surrounding `'`), and it refers to a JSON field; JSON fields do use camel case, and the camel case should be preserved when referring to them.

Reply:
*slaps forehead* Yes, of course! Sorry for adding confusion.